e2e/testcafe-devextreme/tests/chat/chatMessageList.ts

```diff
@@ -0,0 +1,41 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Chat from 'devextreme-testcafe-models/chat';
+import { createUser, generateMessages } from './data';
+import url from '../../helpers/getPageUrl';
+import { createWidget } from '../../helpers/createWidget';
+import { testScreenshot } from '../../helpers/themeUtils';
+import { appendElementTo } from '../../helpers/domUtils';
+
+fixture.disablePageReloads`ChatMessageBox`
+  .page(url(__dirname, '../container.html'));
+
+test('Chat: messagebox', async (t) => {
```

Comment:
```suggestion
test('Empty view', async (t) => {
```

Reply:
This test tests the entire MessageList, not just the EmptyView. I think it should be called Chat: messagelist.

Reply:
[22ab68377f20818332b2142ee35f3be69488a0bb]